cpp/cmake/deps/boost-header-only.patch

```diff
@@ -14,4 +14,3 @@ index 7da31f7..5816de7 100644
 +   //BOOST_ASSERT(0 == ((std::size_t)ret % ::boost::container::dtl::alignment_of<T>::value));
     return static_cast<T*>(ret);
  }
- 
```

Comment:
Think we want to keep this line. Otherwise the patch doesn't apply

Reply:
Reverted in commit ( https://github.com/rapidsai/cucim/pull/474/commits/43606cf60afd5ce2a1693061d684e1da8b5f3687 )